sycl/doc/extensions/experimental/sycl_ext_oneapi_max_work_group_query.md

```diff
@@ -71,9 +71,10 @@ gpu_queue.submit(work_range, ...);
 
 ## Implementation
 
-### Templated queries
+### Deprecated queries
 
-Right now, DPC++ does not support templated device descriptors as they are defined in the SYCL specification section 4.6.4.2 "Device information descriptors". When the implementation supports this syntax, `ext_oneapi_max_work_groups_[1,2,3]d` should be replaced by the templated syntax: `ext_oneapi_max_work_groups<[1,2,3]>`.
+Older versions of DPC++ used information descriptors with the inncorrect namespace; according to SYCL 2020 specification section 6.3.1 "Extension namespace" , `sycl::info::ext_oneapi_max_global_work_groups` is not allowed. 
+Some of those descriptors were also not templated, according to SYCL specification section 4.6.4.2 "Device information descriptors", e.g:  `ext_oneapi_max_work_groups_[1,2,3]d` . These information descriptors are still supported but deprecated.
```

Comment:
It's not really accurate to say that the previous descriptors had an incorrect namespace.  The old descriptors were enums, and they were in the correct namespace given that they were enums.

I think there was a brief period of time when the descriptors were changed to types with the wrong namespace.  However, did we ever release a DPC++ compiler in that form?

Since this is an experimental extension, there is no requirement to document the deprecated forms, but it's OK to do that if you want.  If you want a section on the deprecated APIs, I suggest you just say something like:

> The following descriptors are deprecated and will be removed soon:
>
> * `sycl::info::ext_oneapi_max_global_work_groups` (equivalent to `sycl::ext::oneapi::experimental::info::max_global_work_groups`)
> * `sycl::info::ext_oneapi_max_work_groups_1d` (equivalent to `sycl::ext::oneapi::experimental::info::max_work_groups<1>`

Etc.

There is no need to explain why they were changed.

Reply:
I'm okay with either removing or keeping them. The only reason I deprecated them was because I thought our clients would be using these features

Reply:
> I think there was a brief period of time when the descriptors were changed to types with the wrong namespace. However, did we ever release a DPC++ compiler in that form?

I believe this happened fairly recently, within the ABI-break window, so there should not be any releases with descriptors as types yet.